# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccUtils.h
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/base/nsAccUtils.h
--- a/accessible/base/nsAccUtils.h	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/base/nsAccUtils.h	Tue Jan 04 21:01:36 2022 +0000
@@ -38,17 +38,6 @@
                                int32_t aSetSize, int32_t aPosInSet);
 
   /**
-   * Get default value of the level for the given accessible.
-   */
-  static int32_t GetDefaultLevel(const LocalAccessible* aAcc);
-
-  /**
-   * Return ARIA level value or the default one if ARIA is missed for the
-   * given accessible.
-   */
-  static int32_t GetARIAOrDefaultLevel(const LocalAccessible* aAccessible);
-
-  /**
    * Compute group level for nsIDOMXULContainerItemElement node.
    */
   static int32_t GetLevelForXULContainerItem(nsIContent* aContent);